### ISSUE TITLE
Rubidium 0.7.0 & 0.7.0a Crashes With Oculus

### ISSUE THREAD

**Issue by ChrisLTheNerd** [Asek3/Rubidium#564]
Rubidium mc1.20.1-0.7.0 and mc1.20.1-0.7.0a makes Oculus mc1.20-1.6.4 crash, but works fine with Rubidium mc1.20-0.6.5.

Minecraft: 1.20.1
Forge: 47.1.43

[crash-2023-08-10_17.53.51-client.txt](https://github.com/Asek3/Rubidium/files/12317537/crash-2023-08-10_17.53.51-client.txt)

**Comment by Devilswarchild** [comment]
I can also confirm this as well

**Comment by Dulcemuse** [comment]
Having this problem as well

**Comment by vVenya** [comment]
Having this issue but only after updating my mods, Oculus is crashing the client at bootstrap load with all Rubidium versions compatible for 1.20.1 - Forge loader 47.1.0
For now I'll disable Oculus and Rubidium and play without shaders.

**Comment by Devilswarchild** [comment]
> Having this issue but only after updating my mods, Oculus is crashing the client at bootstrap load with all Rubidium versions compatible for 1.20.1 - Forge loader 47.1.0 For now I'll disable Oculus and Rubidium and play without shaders.

I have been leaving rubidium at the first version released for 1.20.1 and most of us are at 47.1.3 due to waiting on neoforge.

**Comment by vVenya** [comment]
> > Having this issue but only after updating my mods, Oculus is crashing the client at bootstrap load with all Rubidium versions compatible for 1.20.1 - Forge loader 47.1.0 For now I'll disable Oculus and Rubidium and play without shaders.
> 
> I have been leaving rubidium at the first version released for 1.20.1 and most of us are at 47.1.3 due to waiting on neoforge.

Do you think its worth trying to upgrade to mod loader for 47.1.3?
As on my current no Rubidium versions are working with Oculus.

**Comment by Devilswarchild** [comment]
No,  bc the entire dev team split from forge to create neoforge and
recommended everyone to stay on 47.1.3 for the time being

**Comment by Devilswarchild** [comment]
i would try 47.1.3 and see what happens

**Comment by 2663294771** [comment]
是的，我也有一样的问题
，希望尽快适配
